package.json

```diff
@@ -6,6 +6,7 @@
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "4.2.2",
     "govuk-frontend": "^3.6.0",
+    "node-sass": "^4.14.1",
```

Comment:
Just checking the reason for the package.json update?

Reply:
ohh this was accidental. I had to uninstall and re-install node-sass 3 times throughout the day because the scss wasn't rendering - I'll untrack it!